packages/docs/docs/ssi/vp.md

```diff
@@ -4,7 +4,11 @@ sidebar_position: 3
 
 # Verifiable Presentations (VP)
 
-**Verifiable Presentation**, or **VP** for short, expresses data from one or more VCs and is packaged so that the authorship of the data is verifiable. The data in a VP, which is often about the same subject, could have been issued by multiple issuers. Let's say you must prove your University degree, past employment, and date of birth. This data usually originates from different entities, and the owner can combine them to create a single VP.
+**Verifiable Presentation**, or **VP** for short, expresses data from one or more VCs and is packaged so that the authorship of the data is verifiable.
+
+Essentially, VP is a way to verifiably present a Verifiable Credential. VP can contain one or more VCs and is signed using a DIDs.
```

Comment:
present data from Verifiable Credentials ... something like all data or parts of data